lib/archethic_web/graphql_schema.ex

```diff
@@ -58,11 +58,19 @@ defmodule ArchethicWeb.GraphQLSchema do
     """
     field :transaction_chain, list_of(:transaction) do
       arg(:address, non_null(:address))
-      arg(:page, :integer)
+      arg(:paging_address, :string)
```

Comment:
We need to valid as well if it's a valid address, so the type should be `address`